doc/src/manual/arrays.md

```diff
@@ -410,22 +410,43 @@ julia> sum(1/n^2 for n=1:1000)
 1.6439345666815615
 ```
 
-When writing a generator expression with multiple dimensions inside an argument list, parentheses
-are needed to separate the generator from subsequent arguments:
+When writing a generator expression in an argument list, parentheses can be omitted, provided the
+generator is the last argument:
```

Comment:
The suggested version reduces clarity. It simply states when I can omit something, but does not tell me why.
The original version communicates clearly when parentheses are needed and why they are needed (to separate expressions).

Reply:
Yeah you're right. But the second explanation clearly states why they're needed, to separate a generator from subsequent arguments. I don't know how far I can go to tell WHY.

Do you have a suggestion for this?